samples/buildSrc-test-junit5/buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,16 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    testImplementation("org.junit.jupiter:junit-jupiter-api:5.3.2")
+    testRuntime("org.junit.jupiter:junit-jupiter-engine:5.3.2")
+}
+
+tasks.test {
+    useJUnitPlatform()
+}
```

Comment:
This might be more aproprate:

```kotlin
tasks.withType<Test>().configureEach {
    useJunitPlatform()
}
```

It might also be good to demonstrate configuring Junit 5 as well.

For example:
```kotlin
tasks.withType<Test>().configureEach {
    useJunitPlatform {
        filter {
            // Add an example of configuring some filter here.
        }
    }
}
```

Reply:
it seems that _filter_ is not JUnit 5 specific and belongs to _Test_. If i understand it correctly this should be equivalent:
```
tasks.withType<Test>().configureEach {
    filter {
        // Add an example of configuring some filter here.
     }
}
```
I'd suggest to use tags instead, to demonstrate a Junit5 specific configuration:
```
tasks.withType<Test>().configureEach {
    useJUnitPlatform {
        includeTags("fooTag")
    }
}
```

Reply:
You are 100% correct. Good catch.

Reply:
ok, i have changed it